components/activity-section.jsx

```diff
@@ -0,0 +1,41 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+
+var ActivitySection = React.createClass({
+  propTypes: {
+    title: React.PropTypes.string.isRequired,
+    activities: React.PropTypes.array.isRequired
+  },
+  render: function () {
+    return (
+      <section className="activity-section">
+        <h2>
+          Section {this.props.title}
+        </h2>
+        {this.props.activities.map(function (activity, i) {
+          return (
+            <div className="activity" key={i}>
+              <Illustration height={122} width={122}
+              src1x={activity.image1x}
+              src2x={activity.image2x}
+              alt=""
```

Comment:
Alternatively, since the Web Literacy Basics page has the exact same problem, we could just leave it as-is and file a separate issue for it.